src/OneExplorer/OneStorage.ts

```diff
@@ -246,7 +246,9 @@ export class OneStorage {
     try {
       return roots
         .map((root) =>
-          readdirSyncRecursive(root).filter((val) => val.endsWith(".cfg"))
+          readdirSyncRecursive(root).filter(
+            (val) => val.endsWith(".cfg") || val.endsWith(".edgetpucfg")
```

Comment:
How about define list of config file's ext and check with it?
like...
```
const extList = [".cfg", ".edgetpucfg"];
``` 
```suggestion
            (val) => extList.some((ext) => val.endsWith(ext))
```

Reply:
Thank you. I will apply this :-D